cmd/minikube/cmd/root.go

```diff
@@ -62,7 +62,7 @@ var RootCmd = &cobra.Command{
 		}
 
 		logDir := pflag.Lookup("log_dir")
-		if !logDir.Changed {
+		if logDir.Value.String() == "" {
```

Comment:
It seems in other parts of the code we check this via:

```go
if cmd.Flags().Changed(containerRuntime) {}
```

could we do the same here to follow convention?

Reply:
follow the convention,the same problem will still occur, unless you modify cmd.Flags().Changed(), but this impact may be significant